src/main/java/frc/robot/RobotContainer.java

```diff
@@ -107,7 +108,10 @@ public Command buildAutoCommand(String pathName) {
      */
     private void configureAutoCommands() {
         this.commandChooser.setDefaultOption("None", null);
-        this.commandChooser.addOption("Red Three", buildAutoCommand("Red Three"));
```

Comment:
Bonus points if you can figure out how to get the command chooser to add all of the options automatically.

The files are stored in the `deploy/pathplanner` folder.

Reply:
Use `Filesystem.getDeployDirectory()` to get the deploy directory. 

Work from there.